src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -53,30 +53,30 @@ static class BitOperations
         /// </summary>
         /// <param name="value">The value.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool IsPow2(int value) => value >= 0 && (value & (value - 1)) == 0;
+        public static bool IsPow2(int value) => (value & (value - 1)) == 0 && value >= 0;
```

Comment:
I think this needs to be `&& (value > 0)` because `((value & (value - 1)) == 0)` will be `true` for `0` (hence why we have `&& (value != 0)` for the unsigned variant)